src/components/LectureEventModal/LectureEventModal.tsx

```diff
@@ -83,6 +85,18 @@ const LectureEventModal = ({
             </div>
           ))}
         </div>
+        {lectureEvent.title.toLowerCase().includes('anglų kalba') && (
```

Comment:
I would recommend to check only if the word "anglų" is included in the title, if in the future VU will change the titles to be "anglų k." it will still work